### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by He-Pin**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -41,13 +40,13 @@ lazy val common = crossProject.
...
       // "com.typesafe.akka" %% "akka-actor" % "2.5.6"
-      "com.typesafe.akka" %% "akka-actor" % "2.5.12"
+      "com.typesafe.akka" %% "akka-actor" % "2.5.13"
       // "com.typesafe.akka" %% "akka-actor" % "2.5-SNAPSHOT"
     )
   )
   .jsSettings(
     libraryDependencies ++= Seq(
-      "org.akka-js" %%% "akkajsactor" % "1.2.5.12-PERF"
+      "org.akka-js" %%% "akkajsactor" % "1.2.5.12"
```
Comment (andreaTP) [inline build.sbt]: this should become `1.2.5.13` as well :-)

**Comment by andreaTP** [comment]
apart from a minor comment sounds good, I have to take some time to re-run the benchs :+1:

File: build.sbt (hunk 1 above)
Comment (He-Pin) [reply build.sbt]: Lol，I forgot it:).

**Comment by andreaTP** [comment]
thanks!

**Pull request merged**